src/flow/assembly_mh.hh

```diff
@@ -425,7 +426,7 @@ protected:
             double val_ele =  1.;
             static_cast<LinSys_BDDC*>(ad_->lin_sys)->
                             diagonal_weights_set_value( loc_system_.row_dofs[loc_ele_dof], val_ele );
-        }
+		}
```

Comment:
Do not commit whitespace changes (possibly adjust settings of your text editor to work correctly with tabs and spaces).